### PR TITLE
Privacy policy

### LINKED ISSUE #1906: Privacy Policy
Adding a privacy policy to the app, so that the app uses the GET_ACCOUNTS permission

TASKS:

- [X] [GIT] Create branch in Android repo
- [X] [DEV] Add a new option in Settings view
- [X] [DEV] Check branding options
- [X] [DEV] Create a new View for showing the Privacy Policy
- [X] [DEV] Point to https://owncloud.com/privacy-legal/ 
- [X] [DEV] Show web page loading progress
- [X] [DEV] Enable zoom but hide display zoom controls
- [X] [QA] Design test plan
- [X] [QA] Validate test plan @jesmrec
- [ ] [DEV] Code review
- [ ] [GIT] Merge branch in Android repo

### PR DESCRIPTION
**Pull request by davigonz**
Fix #1906

### CODE REVIEW

**Comment by davigonz** [comment]
@jesmrec , @jabarros this issue is fixed, pending to review and pass QA after implementing the corresponding ownBrander part.

**Comment by jesmrec** [comment]
Approved 👍 

CC @davigonz

File: src/com/owncloud/android/ui/activity/PrivacyPolicyActivity.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                    mProgressBar.setVisibility(View.GONE);
+                }
+            }
+        });
+        webview.setWebViewClient(new WebViewClient() {
+            public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
+
+                Snackbar snackbar = Snackbar.make(
+                        findViewById(android.R.id.content),
+                        "An error has occurred: " + description,
```
Comment (jabarros) [inline src/com/owncloud/android/ui/activity/PrivacyPolicyActivity.java]: It should be in strings file to be internationalize

**Comment by jabarros** [comment]
Reviewed

File: src/com/owncloud/android/ui/activity/PrivacyPolicyActivity.java (hunk 1 above)
Comment (davigonz) [inline src/com/owncloud/android/ui/activity/PrivacyPolicyActivity.java]: Sure

**Pull request merged**